tuichain/api/models.py

```diff
@@ -31,6 +32,7 @@ def to_dict(self, private=False):
             result['zip_code'] = self.zip_code
             result['city'] = self.city
             result['id_number'] = self.id_number
+            result['short_bio'] = self.short_bio
```

Comment:
i think short bio should be public, so that other people could see it and better know the requester

also, please fix the conflicts created by the linter and @albertofaria 's changes